pkg/networkmanager/firewall.jsx

```diff
@@ -932,57 +841,47 @@ export class Firewall extends React.Component {
         });
         services.sort((a, b) => a.name.localeCompare(b.name));
 
-        var zones = [...this.state.firewall.activeZones].map(id => {
+        var zones = [...this.state.firewall.activeZones].sort((z1, z2) =>
+            z1 === firewall.defaultZone ? -1 : z2 === firewall.defaultZone ? 1 : 0
+        ).map(id => {
             const zone = this.state.firewall.zones[id];
             zone.name = zone.name || id;
             return zone;
         });
 
         var enabled = this.state.pendingTarget !== null ? this.state.pendingTarget : this.state.firewall.enabled;
-
         return (
-            <div className="container-fluid page-ct">
-                <ol className="breadcrumb">
-                    <li><button role="link" className="link-button" onClick={go_up}>{_("Networking")}</button></li>
-                    <li className="active">{_("Firewall")}</li>
-                </ol>
-                <h1>
-                    {_("Firewall")}
-                    <OnOffSwitch state={enabled}
-                                 disabled={!!this.state.pendingTarget}
-                                 onChange={this.onSwitchChanged} />
-                </h1>
-                <div id="zones-listing">
-                    { enabled && <Listing title={_("Active zones")}
-                             columnTitles={[_("Zone"), "", _("Interfaces"), _("IP Range"), ""]}
-                             emptyCaption={_("No active zones")}
-                             actions={addZoneAction}>
-                        {
-                            zones.map(z => <ZoneRow key={z.id}
-                                                    zone={z}
-                                                    readonly={this.state.firewall.readonly}
-                                                    onRemoveZone={this.onRemoveZone} />)
-                        }
-                    </Listing> }
+            <>
+                <div id="firewall-heading">
+                    <ol className="breadcrumb">
+                        <li><button role="link" className="link-button" onClick={go_up}>{_("Networking")}</button></li>
+                        <li className="active">{_("Firewall")}</li>
+                    </ol>
+                    <h1>
+                        <span>
+                            {_("Firewall")}
+                            <OnOffSwitch state={enabled}
```

Comment:
This has no label, so a screenreader wouldn't know what it enables.
Maybe `aria-label="firewall"` could work.

Reply:
https://dequeuniversity.com/rules/axe/3.2/label?application=AxeChrome